### DISCUSSION TITLE
discussion about opengeos

### DISCUSSION THREAD

**Discussion by slowy07** [opengeos/opengeos.github.io#10] [General]
sorry for brothering, i thinks its be better for this org have a some more discussion via chat, this disscussion talk about opengeos or GIS technology, so keep update about `opengeos` or `GIS` technology

example (microsoft discord community):

![image](https://github.com/user-attachments/assets/3eba4f4f-0f75-474f-99ab-92d292ae6b44)

![image](https://github.com/user-attachments/assets/219f60e4-0d8f-4c51-b2c3-5f2ed942f711)

this discord server talks about microsoft product, share, and answer about microsoft

**Comment by giswqs** [comment]
Yes, it would be great to have, but I don't have much time to manage it. Would you like to create one and help managing it?

**Reply by slowy07** [reply]
yup, i try to create it sir @giswqs

**Reply by slowy07** [reply]
work in progress on server sir @giswqs , you can check here:

https://discord.gg/UQNdA5VHNy

**Comment by slowy07** [comment]
closed cause community was build
